From 5dda4b2393adc996d085d3cf4afddbdee309bb60 Mon Sep 17 00:00:00 2001
From: Divagar Mohandass <divagar.mohandass@intel.com>
Date: Fri, 28 Feb 2020 00:16:03 +0000
Subject: [PATCH 2/2] CHROMIUM: config: enable Intel Jasperlake pinctrl driver

BUG=b:150154277
TEST=Build/deploy and boot the DUT with kernelv5.4
and check pinctrl driver is getting initialized

Change-Id: I17abafa883a82d5a2cd2fa42c31427ef5b6d2d79
Signed-off-by: Divagar Mohandass <divagar.mohandass@intel.com>
---
 chromeos/config/x86_64/chromeos-intel-pineview.flavour.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
index 9452e40ca97c..256cadd18d62 100644
--- a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
+++ b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
@@ -55,6 +55,7 @@ CONFIG_PINCTRL_CANNONLAKE=y
 CONFIG_PINCTRL_CHERRYVIEW=y
 CONFIG_PINCTRL_GEMINILAKE=y
 CONFIG_PINCTRL_ICELAKE=y
+CONFIG_PINCTRL_JASPERLAKE=y
 CONFIG_PINCTRL_SUNRISEPOINT=y
 CONFIG_PINCTRL_TIGERLAKE=y
 # CONFIG_RT2800PCI_RT33XX is not set
-- 
2.25.0.265.gbab2e86ba0-goog

